### PR TITLE
Added mistral models support

### PR DESCRIPTION
**Pull request by rsgrewal-aws**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: introduction-to-bedrock/bedrock_mistral.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import boto3
+import json
+
+#Create the connection to Bedrock
+bedrock = boto3.client(
+    service_name='bedrock',
+    region_name='us-west-2',
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:7]: We should add a comment here saying the mistral is available on us-west-2 only for now

File: introduction-to-bedrock/bedrock_mistral.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+# Let's see all available Meta Models
+
+available_models = bedrock.list_foundation_models()
+for model in available_models['modelSummaries']:
+  if model['providerName']=='Mistral AI':
+    print(json.dumps(model, indent=2))
+
+# Define prompt and model parameters
+# - Mistral Instruct chat template is -- similiar to <s>[INST] Instruction [/INST] Model answer</s>[INST] Follow-up instruction [/INST]
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:23]: I don't think <s></s> is required for bedrock is it?

File: introduction-to-bedrock/bedrock_mistral.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+#- Note that <s> and </s> are special tokens for beginning of string (BOS) and end of string (EOS) while [INST] and [/INST] are regular strings.
+prompt = """<s>[INST]Craft a Python function to convert Celsius to Fahrenheit. If water boils at 100°C, what's that in Fahrenheit?[/INST]</s>"""
+
+body = json.dumps({ 
+	'prompt': prompt,
+    'max_tokens': 200,
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:29]: indentation missing

File: introduction-to-bedrock/bedrock_mistral.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+	'top_p': 0.9,
+	'temperature': 0.2,
+    #'frequency_penalty': 2
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:32]: There is no frequency_penalty parameter in bedrock

File: introduction-to-bedrock/bedrock_mistral.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+})
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:33]: there is a top_k parameter, do we want to add it to the example?

File: introduction-to-bedrock/bedrock_mistral.py [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+name: Person First Name
+lastname: Person Last Name
+address: #1 Samuel St.
+
+Just generate the JSON object without explanations:
+[/INST]"""
+
+body = json.dumps({ 
+	'prompt': prompt,
+    'max_tokens': 200,
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:81]: same comments for the parameters here.. no frequency_penalty and maybe we should add top_k

File: introduction-to-bedrock/bedrock_mistral.py [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+	'top_p': 0.9,
+	'temperature': 0.2,
+    #'frequency_penalty': 2
+})
+
+modelId = 'mistral.mistral-7b-instruct-v0:2'
+#modelId = 'mistral.mixtral-8x7b-instruct-v0:1'
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mistral.py:88]: remove this one as there is a second notebook for it

File: introduction-to-bedrock/bedrock_mixtral.py [hunk 8]
```diff
@@ -0,0 +1,98 @@
+import boto3
+import json
+
+#Create the connection to Bedrock
+bedrock = boto3.client(
+    service_name='bedrock',
+    region_name='us-west-2',
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mixtral.py:7]: add region comment

File: introduction-to-bedrock/create_your_first_bedrock_application/GettingStartedWithAmazonBedrock.ipynb [hunk 9]
```diff
@@ -699,6 +729,50 @@
...
+    "address: #1 Samuel St.\n",
+    "\n",
+    "Just generate the JSON object without explanations:\n",
+    "[/INST]\"\"\"\n",
+    "body = json.dumps({ \n",
+    "\t'prompt': prompt,\n",
+    "    'max_tokens': 200,\n",
+    "\t'top_p': 0.9,\n",
+    "\t'temperature': 0.2,\n",
+    "    #'frequency_penalty': 2\n",
```
Comment (mttanke) [inline introduction-to-bedrock/create_your_first_bedrock_application/GettingStartedWithAmazonBedrock.ipynb:760]: same comment for parameters here

File: introduction-to-bedrock/bedrock_mixtral.py [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+lastname: Person Last Name
+address: #1 Samuel St.
+
+Just generate the JSON object without explanations:
+[/INST]"""
+
+body = json.dumps({ 
+	'prompt': prompt,
+    'max_tokens': 200,
+	'top_p': 0.9,
```
Comment (mttanke) [inline introduction-to-bedrock/bedrock_mixtral.py:81]: same comment for parameters here

**Review by mttanke — CHANGES_REQUESTED** [review]

File: introduction-to-bedrock/bedrock_mistral.py (hunk 1 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:8]: fixed

File: introduction-to-bedrock/bedrock_mistral.py (hunk 2 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:24]: it is good to have a sample of the prompt template -- it is recomended by Mistral

File: introduction-to-bedrock/bedrock_mistral.py (hunk 3 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:30]: this is fine - probably visual . i checked and re commited

File: introduction-to-bedrock/bedrock_mistral.py (hunk 5 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:34]: yes but Mistral models have it -- that is why i left it in comment. it is a matter of time before this comes in

File: introduction-to-bedrock/bedrock_mistral.py (hunk 6 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:82]: fixed

File: introduction-to-bedrock/bedrock_mistral.py (hunk 7 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mistral.py:89]: fixed

File: introduction-to-bedrock/bedrock_mixtral.py (hunk 8 above)
Comment (rsgrewal-aws) [reply introduction-to-bedrock/bedrock_mixtral.py:8]: fixed

**Comment by rsgrewal-aws** [comment]
changes in the new version uploaded

**Review by mttanke — APPROVED** [review]

**Pull request merged**